GoogleSignIn/Sources/GIDSignIn_Private.h

```diff
@@ -44,10 +46,13 @@ typedef void (^GIDDisconnectCompletion)(NSError *_Nullable error);
 /// Redeclare |currentUser| as readwrite for internal use.
 @property(nonatomic, readwrite, nullable) GIDGoogleUser *currentUser;
 
-/// Private initializer for |GIDSignIn|.
-- (instancetype)initPrivate;
+/// Expose `GIDAppCheck` for tests.
```

Comment:
We should avoid needing to expose a property for tests.

See the other comment regarding using composition for `id<GIDAppCheckProvider>` in `GIDSignIn` so you don't need to expose it here.

Reply:
We could put this property in the public header: `GIDSignIn.h`. What do you think? It's `readonly` so it shouldn't be problematic to expose.

Reply:
As a general principle, I'd prefer to keep the public API minimized as much as possible. A `readonly` property seems harmless, but once we make it public, app developers can use it in their ways (such as using it in tests as we do), and we're responsible for maintaining it. That would create additional hurdles for us if we ever need to refactor our implementation. Given that the App Check integration is very new, there's a great chance that we don't get it right the first time and find ourselves needing to refactor. I think it's particularly important to emphasize the need to minimize the public API at this point.

Reply:
Removed and reorganized file structure.